src/Test/Utilities/Desktop/ThrowingTraceListener.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class ThrowingTraceListener : System.Diagnostics.TraceListener
+    {
+        public override void Fail(string message, string detailMessage)
+        {
+            throw new DebugAssertFailureException(string.Format("{0}\r\n{1}", message, detailMessage));
```

Comment:
This was just the code that was already there -- I renamed the file and Git isn't picking up on it. I can update though if you'd like.